mindmeld/ser.py

```diff
@@ -12,36 +12,11 @@
 # limitations under the License.
 
 """This module contains the system entity recognizer."""
-import json
 import logging
-from enum import Enum
-
-import pycountry
-
-from .components.request import validate_language_code, validate_locale_code
-from .core import Entity, QueryEntity, Span, _sort_by_lowest_time_grain
-from .exceptions import SystemEntityResolutionError
-from .system_entity_recognizer import SystemEntityRecognizer
+from .system_entity_recognizer import DucklingRecognizer
 
 logger = logging.getLogger(__name__)
 
-SUCCESSFUL_HTTP_CODE = 200
-
-
-class DucklingDimension(Enum):
-    AMOUNT_OF_MONEY = "amount-of-money"
-    DISTANCE = "distance"
-    DURATION = "duration"
-    NUMERAL = "numeral"
-    ORDINAL = "ordinal"
-    QUANTITY = "quantity"
-    TEMPERATURE = "temperature"
-    VOLUME = "volume"
-    EMAIL = "email"
-    PHONE_NUMBER = "phone-number"
-    URL = "url"
-    TIME = "time"
-
 
 def get_candidates(
```

Comment:
Why cant `get_candidates ` be a class method of `DucklingRecognizer ` so that we can just access the recognizer through `self` instead of instantiate the singleton again here?

Reply:
get_candidates is already refactored into DucklingRecognizer. This is maintained here as a deprecated method.

Reply:
Can you annotated these functions as deprecated to that we can remove them in the near future?